src/libtiled/mapreader.cpp

```diff
@@ -716,8 +723,26 @@ void MapReaderPrivate::readTilesetWangSets(Tileset &tileset)
             const QString name = atts.value(QLatin1String("name")).toString();
             const WangSet::Type type = wangSetTypeFromString(atts.value(QLatin1String("type")).toString());
             const int tile = atts.value(QLatin1String("tile")).toInt();
+            const bool randomizeOrientation = atts.value(QLatin1String("randomizeOrientation")).toInt()!=0;
```

Comment:
I've kept all attributes so far completely lowercase, please also use this style for new attributes.

Reply:
:+1: